src/modules/balance/ModuleTokensValue.vue

```diff
@@ -73,7 +73,9 @@ export default {
       return formatFiatValue(this.totalTokenFiatValue).value;
     },
     tokenImages() {
-      const firstFive = this.tokensList.slice(0, 5);
+      const filterNoImageTokens = this.tokensList.filter(token => token.img);
```

Comment:
```
tokenImages() {
      return this.tokensList
        .filter(token => token.img)
        .slice(0, 5)
        .map(item => {
          return item.img;
        });
    },
```
Might not need variable names since the functions are clear, but up to you :smiley:

Reply:
@andrewdahn can you change it to this ^